src/components/ErrorFallback/ErrorFallback.tsx

```diff
@@ -0,0 +1,31 @@
+import { useErrorBoundary } from "react-error-boundary"
+
+import "./ErrorFallback.css"
+
+import arebyte from "data-base64:assets/arebyte-Plugin-blue.png"
+
+const ErrorFallBack = ({ error }) => {
+  const { resetBoundary } = useErrorBoundary()
+
+  return (
+    <main className="error-page">
+      <img
+        src={arebyte}
+        alt="Arebyte Plugin Logo"
+        className="error-logo"
+      />
+      <div className="margin-top-lg stack error-message">
+        <p>Something went wrong:</p>
+        <p style={{ color: "red" }}>{error.message}</p>
+        <button
```

Comment:
Maybe we want to move this into the stylesheet? It's a minimal change but in future if we want to change it for any reason it would make sense to just open the `.css` file.

Also a nice chance to use the `nth-child` pseudo class selector :P

Reply:
Yes good call. I did it to be quick and dirty and then forgot :P